### ISSUE TITLE
ReleaseBinaries.java:47-49: Implement attaching artifacts to release,...

### ISSUE THREAD

**Issue by alex-palevsky** [yegor256/rultor#832]
Puzzle `662-8cc0c29a` in `src/main/java/com/rultor/agents/github/ReleaseBinaries.java:47-49` has to be resolved: Implement attaching artifacts to release, remove PMD suppression below and enable attachesBinariesToRelease test. Similar functionality exists in CommentsTag so you can look there for clues.... The puzzle was created by krzyk on 7-Feb-2015.

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by alex-palevsky** [comment]
@yegor256 attached this issue to milestone "0.3" (let me know if this is wrong)

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear OK, I will try to assign someone else

**Comment by original-brownbear** [comment]
@alex-palevsky this is not postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is not postponed.

@original-brownbear sure, thanks, I removed "postponed" tag to it

**Comment by original-brownbear** [comment]
@alex-palevsky assign this to me please.

**Comment by alex-palevsky** [comment]
> @alex-palevsky assign this to me please.

@original-brownbear of course, the task is yours, go ahead

**Comment by original-brownbear** [comment]
@alex-palevsky this needs to wait for #1060.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this depends on #1060.

@original-brownbear yep, got it, let's wait for #1060 being solved

**Comment by original-brownbear** [comment]
@alex-palevsky this is postponed.

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear thanks, I added "postponed" label

**Comment by alex-palevsky** [comment]
> @alex-palevsky this is postponed.

@original-brownbear right, I'll try to find someone else for this task

**Comment by alex-palevsky** [comment]
@original-brownbear Finished, thanks, closing..
